provider/Cargo.toml

```diff
@@ -34,3 +40,7 @@ databake = { version = "0.2.0", features = ["derive"], optional = true }
 yoke = { version = "0.8.0", features = ["derive"], optional = true }
 serde_json = { version = "1.0.140", optional = true }
 
+# ZoneInfo64 dependencies
+resb = { git = "https://github.com/unicode-org/icu4x.git", optional = true }
```

Comment:
We should cut a release; make a PR to ICU4X that finalizes the resb Cargo.toml (make it publish = false), write a changelog entry, and ping me for a release.